### DISCUSSION TITLE
部品の交換、購入は可能でしょうか？

### DISCUSSION THREAD

**Discussion by nepiany** [Yowkees/keyball#176] [Q&A] [answered]
一通り組み立てが完了したのですがトラックボールだけが動作しませんでした。

この辺りは確認しましたが解決しませんでした。
https://github.com/Yowkees/keyball/discussions/51#discussioncomment-1044412

トラックボール用の回路自体が認識されていないのか、REMAPに接続しても初期状態でボールなしレイアウトになってしまいます。
回路図を見ながらテスターで導通確認もしたのですが、 Pro Micro からトラックボール用のL字コンスルーまでは問題ありませんでした。

キー数違いでもう1セット購入しており、そちらのトラックボール用基盤と交換したところ、問題なく認識・動作しました。
動かなかった方の基盤もはんだ付け等特にミスはなかったと思うのですが…

トラックボール用のセンサーと基盤の交換いただく、または購入することは可能でしょうか？

また、それとは別にTRSジャックも一つ追加で購入したいのですが可能でしょうか？
これに関しては完全にこちらのミスで、裏表を間違えて取り付け、外すときに壊してしまいました…
（これももう一方の部品を使いひとまずはなんとかなりました）

お手数をおかけしますがよろしくお願いいたします。

**Comment by Yowkees** [comment, marked as answer]
トラックボール基板を入れ替えた所動作したということで、トラックボール基板の破損が濃厚ですね。
トラックボール基板とセンサー・レンズのセットで販売可能です。
詳細をご連絡しますので、白銀ラボホームページのお問合せからメールをいただけますでしょうか？
ご連絡お待ちしております。

**Reply by nepiany** [reply]
問い合わせさせていただきました。
よろしくお願いいたします。
